icaruscode/Decode/ChannelMapping/ChannelMapPostGres_tool.cc

```diff
@@ -302,7 +379,8 @@ int ChannelMapPostGres::BuildTPCFragmentIDToReadoutIDMap(TPCFragmentIDToReadoutI
 const unsigned int CHANNELSPERBOARD = 64;
 int ChannelMapPostGres::BuildTPCReadoutBoardToChannelMap(TPCReadoutBoardToChannelMap& rbChanMap) const
 {
-    const std::string  name("icarus_hardware_prd");
+    assert(fCurrentTable);
+    const std::string  name(fCurrentTable->TPCreadoutBoardMap);
     const std::string  dburl("https://dbdata0vm.fnal.gov:9443/QE/hw/app/SQ/query?dbname=icarus_hardware_prd");
     const std::string  dataType("daq_channels");
```

Comment:
I'm a bit confused by the usage of `name `and `dataType`. The variable that is actually being used as the table name in the URL query (`?t=`) further down in `GetDataset()` is `dataType`. My feeling is that `name` is a just a label required by `getDataWithTimeout()`.